lib/launch-fastify.js

```diff
@@ -103,13 +103,15 @@ async function launchFastify(file, options, address) {
   const logger = customLogger(mergedOptions, options)
   const fastify = fastifyBuilder({ ...exportFastifyOptions(mergedOptions), logger })
 
+  let swaggerSpecification = process.env.SWAGGER_SPECIFICATION || 'openapi'
```

Comment:
The specification is the `openapi` one, not the `swagger` one, maybe we can call the variable `OPENAPI_SPECIFICATION`?

@davidebianchi @fredmaggiowski what do you think about that?

Reply:
Yes, it is better to use `OPENAPI_SPECIFICATION`

Reply:
Maybe, instead of a string between `swagger` and `openapi`, should we use the version to expose?

Reply:
Ok, I'll change the variable name.
About the openapi version, reading [this section](https://github.com/fastify/fastify-swagger#options) from the `fastify-swagger` documentation, it says that specifying the `openapi` key, the used version is the v3.0.3